features/bootstrap/DatasetViewContext.php

```diff
@@ -416,7 +416,12 @@ public function iHaveSetPagesizeToOn($arg1, $arg2)
         }
     }
 
-
-
-
+    /**
+     * @Then There should be a meta-tag with name :arg1 and content :arg2
+     */
+    public function thereShouldBeMetaTagWithNameAndWithContent($arg1, $arg2)
```

Comment:
Maybe this can be shortened, e.g. `@Then the page meta :arg1 should be :arg2`?

Reply:
Change it to `There is a meta tag :arg1  with DOI :arg2`.